lib/attr_default/railtie.rb

```diff
@@ -1,3 +1,5 @@
+# frozen_string_literal: true
+
 module AttrDefault
   # Rails 3 initialization
```

Comment:
Now that we're locking this gem to use Rails >= 4, we can remove any Rails 3 specific code. Not sure that's worth doing as part of this effort, though...

Reply:
Hm would that mean just getting rid of the railtie file or is there more to it?

Reply:
I'm not sure what it would mean because I don't know the ins-and-outs of all the code here. 🙂 

Assuming we have 100% test coverage that exercises the integration points with Rails 4, we could remove code that looks like Rails 3 and rely on the tests to tell us if those removals. are ok.